tasks/internal_reconcile_report_phantom/requirements-dev.txt

```diff
@@ -0,0 +1,11 @@
+## Libraries needed for testing
+coverage==5.3
+pytest==6.1.2
+
+## Application libraries needed for testing
+psycopg2-binary==2.8.6
+
+## Application libraries
+cumulus-message-adapter-python==2.0.0
+fastjsonschema~=2.15.1
+../../shared_libraries[database, reconcile]
```

Comment:
Missing additional validation libraries. See get_current_archive_list for example of this.

Reply:
added the missing ones